Project.toml

```diff
@@ -3,10 +3,23 @@ uuid = "03207cf0-e2b3-4b91-9ca8-690cf0fb507e"
 authors = ["Alexandr Romanenko <waralex@gmail.com>"]
 version = "0.2.0"
 
+[weakdeps]
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
+
+[extensions]
+DashBasePlotsExt = "Plots"
+
 [deps]
 JSON3 = "0f8b85d8-7281-11e9-16c2-39a750bddbf1"
+
+[extras]
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 
 [compat]
 JSON3 = "1"
-julia = "1.2"
+julia = "1.9"
```

Comment:
I don't think we want this here.

I think it's fine to have `DashBasePlotsExt.jl` only work for Julia 1.9+, but Dash.jl should still work for [Julia 1.6 and up](https://github.com/plotly/Dash.jl/blob/d01ce73b4667991f3441e167816fd00a24a35a54/Project.toml#L41).

Reply:
Thats will be the case. Users of julia 1.6 will get Dash with DashBase v0.2.0 while users on 1.9+ will get DashBase v0.2.1 with the extension

Reply:
I think it might be better to setup a https://github.com/cjdoris/PackageExtensionCompat.jl flow and keep our current `julia` compat than having to possibly backport fixes to  DashBase `v0.2.x`.

We'll need to setup  https://github.com/cjdoris/PackageExtensionCompat.jl to implement the PlotlyJS extension (part of https://github.com/plotly/Dash.jl/issues/221) anyway.